### ISSUE TITLE
Transfer the method to SD3, I encountered some confusion on train stage2

### ISSUE THREAD

**Issue by LeonNerd** [ToTheBeginning/PuLID#198]
I am also trying to train the model, but I have some confusion regarding the loss design in the second stage. According to the paper, the lighting branch generates a prediction image from pure noise in 4 steps and calculates the ID loss with the original image. This process is not influenced by the diffusion loss branch's tempstep, but rather it is affected only by the most recently updated pulid encoder and pulid_ca models. By adding these two losses together for backpropagation, does the ID loss indirectly affect the gradient updates? Currently, I am quite confused as the model is converging very slowly.I would like to know how the ID loss should be correctly added to the training process

**Comment by LeonNerd** [comment]
@ToTheBeginning @zsxkib @guozinan126
